src/main/webapp/app/converter/converter.component.html

```diff
@@ -43,7 +43,7 @@
             </div>
             <div class="col-md-7">
                 <button class="btn btn-primary button-margin" (click)="uploadFiles()" *ngIf="filesToUpload.length > 0">Upload File(s)</button>
-                <span *ngIf="uploadMessage.content.length > 0" [style.color]="uploadMessage.color"> {{uploadMessage.content}} </span>
+                <span *ngIf="uploadMessage['content'].length > 0" [style.color]="uploadMessage.color"> {{uploadMessage.content}} </span>
```

Comment:
May I know why is this change necessary? It is the same thing

Reply:
When I build war file, it cannot detect content object since content is an inner object of uploadMessage. To fix this error, I access inner object by name.

Reply:
Honestly, I still didn't get it. object.keyName is exactly the same with object['keyName'] right?

Reply:
Yes. However, it failed to build war file when I use object.keyName. I guess the inner objects are not initialized when build war file.